controllers/users.js

```diff
@@ -24,7 +24,11 @@ const verifyUser = async (req, res) => {
     logger.error(`Error while verifying user: ${error}`);
     return res.boom.serverUnavailable(SOMETHING_WENT_WRONG);
   }
-  fetch(process.env.IDENTITY_SERVICE_URL, "POST", null, { userId }, { "Content-Type": "application/json" });
+  fetch(process.env.IDENTITY_SERVICE_URL, {
+    method: "POST",
+    body: { userId },
+    headers: { "Content-Type": "application/json" },
+  });
   return res.json({
```

Comment:
right, thanks for pointing out

Reply:
I haven't worked on this API so I think this issue shouldn't be addressed in this PR, it might break something I am not aware of

Reply:
I believe @bhtibrewal concern is that if we modify this code, it could potentially impact the frontend code that consumes this API. Specifically, there might be code in the frontend that checks the status or relies on this specific implementation, and any changes made here could potentially impact or break the code there.

Reply:
Okay, sounds good, but we need to take such things in account. No worries @bhtibrewal